scripts/rmc_flags.py

```diff
@@ -0,0 +1,103 @@
+    add_flag(group, "--quiet", "-q", action="store_true",
+        help="Produces no output, just an exit code and requested artifacts. Overrides --verbose")
+
+def add_um_flags(make_group, add_flag):
+    group = make_group("Um flags", "Um") # TODO: Come up with name and description
+    add_flag(group, "--c-lib", nargs="*", default=[], action="extend",
+        help="External C files referenced by Rust code")
+    add_flag(group, "--function", default="main",
+        help="Entry point for verification")
+
+def add_artifact_flags(make_group, add_flag):
+    group = make_group("Artifact flags", "Produce artifacts in addition to a basic RMC report.")
+    add_flag(group, "--target-dir", default=".",
+        help="Directory for all generated artifacts")
+    add_flag(group, "--keep-temps", action="store_true",
+        help="Keep temporary files generated throughout RMC process")
+    add_flag(group, "--gen-c", action="store_true",
+        help="Generate C file equivalent to inputted program")
+    add_flag(group, "--gen-symbols", action="store_true",
+        help="Generate a symbol table")
+
+def add_check_flags(make_group, add_flag):
+    group = make_group("Check flags", "Disable some or all default checks.")
+    add_flag(group, "--no-default-checks", action="store_true", help="Disable all default checks")
+    add_flag(group, "--no-memory-safety-checks", action="store_true", help="Disable default memory safety checks")
+    add_flag(group, "--no-overflow-checks", action="store_true", help="Disable default overflow checks")
+    add_flag(group, "--no-unwinding-checks", action="store_true", help="Disable default unwinding checks")
+
+def add_visualizer_flags(make_group, add_flag):
+    group = make_group("Visualizer flags", "Generate an HTML-based UI for the generated RMC report.")
+    add_flag(group, "--srcdir", default=".") # TODO: help?
+    add_flag(group, "--wkdir", default=".") # TODO: help?
+    add_flag(group, "--visualize", action="store_true",
+        help="Generate visualizer report; open report/html/index.html")
+
+def add_other_flags(make_group, add_flag):
+    group = make_group("Other flags")
+    add_flag(group, "--allow-cbmc-verification-failure", action="store_true",
+        help="Do not produce error retcode on CBMC verification failure")
+    add_flag(group, "--mangler", default="v0",
+        help="Change what mangler is used by the Rust compiler")
+    add_flag(group, "--dry-run", action="store_true", 
+        help="Print commands instead of running them")
+
+def add_developer_flags(make_group, add_flag):
+    group = make_group("Developer flags", "These are generally meant for use by RMC developers, and are not necessarily stable.")
+    add_flag(group, "--cbmc-args", nargs=argparse.REMAINDER, default=[], 
+                        help="Pass through directly to CBMC; must be the last flag")
+
+def add_flags(
+    parser, 
+    exclude_flags=[],
+    exclude_groups=[]):
```

Comment:
```suggestion
def add_flags(parser, exclude_flags=[], exclude_groups=[]):
```
Consider using a formatter (like `autopep`)